go/vt/vttablet/tabletserver/throttle/throttler_test.go

```diff
@@ -7,13 +7,53 @@
 package throttle
 
 import (
+	"context"
+	"fmt"
+	"sync/atomic"
 	"testing"
 	"time"
 
 	"github.com/patrickmn/go-cache"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"vitess.io/vitess/go/vt/topo"
+	"vitess.io/vitess/go/vt/vttablet/tabletserver/throttle/config"
+	"vitess.io/vitess/go/vt/vttablet/tabletserver/throttle/mysql"
+
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 )
 
+type FakeTopoServer struct {
+}
```

Comment:
FWIW, you could probably use the existing `faketopo` package if it's easier.

Reply:
I tried but it did not make much sense to me. I need a specific fake behavior for specific functions?